Practice3/Decision1.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practice3
+{
+    class Decision1
+    {
+        List<int> _list = new List<int>();
+
+        public int MinRange { get; set; }
+        public int MaxRange { get; set; }
+
+        public void SetRange()
```

Comment:
as I can see, this task is not implemented. could you please fix it?

![image](https://user-images.githubusercontent.com/9382934/113508855-085ff100-955b-11eb-9d5d-8f8e7b4ca67a.png)

Reply:
Done MaxRange must be at least MinRange+10.
However, for the decision1 a user can add as many values as s(he) wants even if the array range is between o and 1